components/rendering/src/markdown.rs

```diff
@@ -16,7 +16,7 @@ use utils::site::resolve_internal_link;
 use context::RenderContext;
 use table_of_contents::{make_table_of_contents, Header, TempHeader};
 
-const CONTINUE_READING: &str = "<p><a name=\"continue-reading\"></a></p>\n";
+const CONTINUE_READING: &str = "<p id="continue-reading"><a name=\"continue-reading\">continue reading...</a></p>\n";
```

Comment:
adding an id is fine (although I would call it `zola-continue-reading` but the text is a bad idea, not everyone is making a site in English.
I think a better way would be to have a template for it so users can customise it as much as they want

Reply:
Yea I wasnt too sure about the text. Let me make this PR to only add the id (which is actually blocking me) and figure out the text via template in a different PR or create an issue.

I also need to update some tests.